zerver/lib/streams.py

```diff
@@ -113,6 +115,7 @@ def create_streams_if_needed(realm: Realm,
             realm,
             stream_dict["name"],
             invite_only=stream_dict.get("invite_only", False),
+            is_web_public=stream_dict.get("is_web_public", False),
```

Comment:
So the test included in this commit doesn't actually test this code path; is that intentional?